websites/P/Pandora/presence.ts

```diff
@@ -1,85 +1,87 @@
 const presence = new Presence({
-    clientId: "608109837657702566"
-  }),
-  strings = presence.getStrings({
-    play: "presence.playback.playing",
-    pause: "presence.playback.paused"
-  });
+  clientId: "608109837657702566"
+}),
+strings = presence.getStrings({
+  play: "presence.playback.playing",
+  pause: "presence.playback.paused"
+});
 
 function stripText(element: HTMLElement, id = "None", log = true) {
   if (element && element.firstChild) return element.firstChild.textContent;
   else {
     if (log) {
-      presence.info(
-        `%cPandora%cERROR%c An error occurred while stripping data off the page. Please contact Alanexei on the PreMiD Discord server, and send him a screenshot of this error. ID: ${id}`
+      presence.error(
+        `An error occurred while stripping data off the page. Please contact Alanexei on the PreMiD Discord server, and send him a screenshot of this error. ID: ${id}`
```

Comment:
Alanexei is not in PreMiD server anymore, so just either tell them to create an issue on github or contact you

Reply:
Okay, I'd be more than willing to maintain this one so I'll change it to me